package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "discipl-core-irma",
```

Comment:
Naming should be `@discipl/core-irma`. This means its published in scope discipl.

Reply:
Fixed.